projects/scion/microfrontend-platform/src/lib/client/connect-options.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2018-2020 Swiss Federal Railways
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ *  SPDX-License-Identifier: EPL-2.0
```

Comment:
There seems to be an extra space in every copyright header

Reply:
Fixed indentation in all copyright headers